airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -106,6 +106,9 @@ definitions:
         type: string
       state:
```

Comment:
We should probably add the following to the AirbyteStreamState object:
```
required:
  - name
  - state
```
to make it clear that those two properties are always required, and `namespace` is optional

Reply:
Done